libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/ModelOverview.tsx

```diff
@@ -172,7 +168,9 @@ export class ModelOverview extends React.Component<
     ) {
       defaultSelectedMetrics = [
         QuestionAnsweringMetrics.ExactMatchRatio,
-        QuestionAnsweringMetrics.F1Score
```

Comment:
We dont want to remove F1Score from here

Reply:
I thought it's not an interesting default metric, and we don't want too many, so that's why I removed it... user can still add it in.  None of the other tasks have this many default metrics.